yarn.lock

```diff
@@ -1731,6 +1745,13 @@
   dependencies:
     jsbi "^4.1.0"
 
+"@ckb-lumos/bi@0.21.0-next.2":
```

Comment:
Multiple versions of lumos are introduced

1. @ckb-lumos/bi@0.20 at line 1735
2. @ckb-lumos/bi@0.21.0-next.1 at line 1742
3. @ckb-lumos/bi@0.21.0-next.2 at line 1748

Reply:
Lumos's version is unified and fixed into the same one